### ISSUE TITLE
Not working on current version of Odoo?

### ISSUE THREAD

**Issue by mochsner** [razib-hossen/ERP-17#7]
Thanks for gathering this code from the 17.0 Odoo tutorial. It has been helpful just to poke around, as I don't have the time to actually code everything, but need to have something for reference ( I am surprised odoo.com doesn't offer this code for us, but that's besides the point ...)

Since I'm now trying to run this, to do my own Odoo development on top of it, I figure I'd throw this error in here. 

Basically, from following the steps, below occurs:
![image](https://github.com/user-attachments/assets/9898f2f3-d29d-48c3-8c3c-49d3ec6bacf0)

Will hopefully figure it out over the next few days and comment. Maybe it's something really silly that I'm missing -- if it's not obvious, I am brand new to odoo :)

**Comment by mochsner** [comment]
Removing this action from `property_view.xml` is my workaround "fixed", haha.
![image](https://github.com/user-attachments/assets/782e09db-813e-43b0-9d16-86eadf98232e)
